day7/will/index.html

```diff
@@ -0,0 +1,172 @@
+    <div class="mobileview-container">
+      <img src="logo.png" alt="logo" />
+      <p>Saturn is best experienced on our mobile app</p>
+    </div>
+    <button>Download Saturn</button>
+  </section>
+  <section class="tabletview-wrapper">
+    <div class="tabletview-title-container">
+      <img src="logo2.png" alt="logo" />
+      <div class="tabletview-title-block">
+        <div class="tabletview-title-circle">
+          <p>Period 2</p>
+          <h2>45 : 27</h2>
+          <h4>25% done</h4>
+        </div>
+        <div class="tabletview-title-text">
+          <div class="tabletview-title-classroom">
+            <h3>AP Computer Science</h3>
+            <p>Mr.Nessi • Room 25A • 9:30 - 10:43</p>
+          </div>
+          <div class="tabletview-title-progress">
+            <p><span>25%</span> of your day is completed!😎</p>
+            <div class="tabletview-progressbar">
+              <div></div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="tabletview-content-container">
+      <div class="tabletview-content-title">
+        <div class="tabletview-content-title-text">
+          <p>Belmont High School</p>
+          <h2>Monday, Feb. 24 - <span>A Day</span></h2>
+        </div>
+        <button><i class="fas fa-sign-out-alt"></i>Log Out</button>
+      </div>
+      <div class="tabletview-content-list-block">
+        <div class="tabletview-content-list">
+          <h3>1</h3>
+          <div class="tabletview-content-list-text">
+            <h4>Philosophy</h4>
+            <p>8:00-9:31 • Mr. Nessi • Room 25A</p>
+          </div>
+        </div>
+        <div class="tabletview-content-list">
+          <h3>2</h3>
+          <div class="tabletview-content-list-text">
+            <h4>Philosophy</h4>
+            <p>8:00-9:31 • Mr. Nessi • Room 25A</p>
+          </div>
+        </div>
+        <div class="tabletview-content-list">
+          <h3>3</h3>
+          <div class="tabletview-content-list-text">
+            <h4>Philosophy</h4>
+            <p>8:00-9:31 • Mr. Nessi • Room 25A</p>
+          </div>
+        </div>
+        <div class="tabletview-content-list">
+          <h3>4</h3>
+          <div class="tabletview-content-list-text">
+            <h4>Philosophy</h4>
+            <p>8:00-9:31 • Mr. Nessi • Room 25A</p>
+          </div>
+        </div>
+        <div class="tabletview-content-list">
+          <h3>5</h3>
+          <div class="tabletview-content-list-text">
+            <h4>Philosophy</h4>
+            <p>8:00-9:31 • Mr. Nessi • Room 25A</p>
+          </div>
+        </div>
+        <div class="tabletview-content-list">
+          <h3>6</h3>
+          <div class="tabletview-content-list-text">
+            <h4>Philosophy</h4>
+            <p>8:00-9:31 • Mr. Nessi • Room 25A</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="laptop-wrapper">
```

Comment:
use only 1 set of html for tablet and laptop

Reply:
adjusted